protocol/x/clob/keeper/liquidations.go

```diff
@@ -222,34 +224,49 @@ func (k Keeper) PlacePerpetualLiquidation(
 
 	// TODO(DEC-1323): Potentially allow liquidating the same perpetual + subaccount
 	// multiple times in a block.
+	perpetualId := liquidationOrder.MustGetLiquidatedPerpetualId()
 	k.MustUpdateSubaccountPerpetualLiquidated(
 		ctx,
 		liquidationOrder.GetSubaccountId(),
-		liquidationOrder.MustGetLiquidatedPerpetualId(),
+		perpetualId,
 	)
 
-	telemetry.IncrCounter(
-		1,
-		metrics.Liquidations,
-		metrics.PlacePerpetualLiquidation,
-		metrics.Count,
-	)
+	labels := []gometrics.Label{
+		metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+		metrics.GetLabelForBoolValue(metrics.IsBuy, liquidationOrder.IsBuy()),
+	}
 
+	// Stat the number of liquidation orders placed.
 	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(liquidationOrder.GetBaseQuantums().ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
+		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Count},
+		1,
+		append(
+			[]gometrics.Label{
+				metrics.GetLabelForBoolValue(
+					metrics.Unfilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == 0,
+				),
+				metrics.GetLabelForBoolValue(
+					metrics.FullyFilled,
+					orderSizeOptimisticallyFilledFromMatchingQuantums == liquidationOrder.GetBaseQuantums(),
+				),
+			},
+			labels...,
+		),
 	)
 
-	telemetry.IncrCounterWithLabels(
-		[]string{metrics.Liquidations, metrics.PlacePerpetualLiquidation, metrics.Filled, metrics.BaseQuantums},
-		metrics.GetMetricValueFromBigInt(orderSizeOptimisticallyFilledFromMatchingQuantums.ToBigInt()),
-		[]gometrics.Label{
-			metrics.GetLabelForIntValue(metrics.PerpetualId, int(liquidationOrder.MustGetLiquidatedPerpetualId())),
-		},
-	)
+	// Stat the volume of liquidation orders placed.
```

Comment:
to confirm, there is a metric for liquidations FILLED somewhere else right?

Reply:
This metric has `Filled/PartiallyFilled/Unfilled` as label, meant to be a simple count.

I added a separate metric in `process_single_match` for the actual filled quote quantums.